torch/nn/modules/transformer.py

```diff
@@ -178,8 +178,10 @@ def forward(self, src: Tensor, tgt: Tensor, src_mask: Optional[Tensor] = None, t
 
             Note: [src/tgt/memory]_mask ensures that position :math:`i` is allowed to attend the unmasked
             positions. If a BoolTensor is provided, positions with ``True``
-            are not allowed to attend while ``False`` values will be unchanged. If a FloatTensor
-            is provided, it will be added to the attention weight.
+            are not allowed to attend while ``False`` values will be unchanged.
+            (This definition is opposite to that of :attr:`attn_mask` in
```

Comment:
Could you move this to a `..note::` above to make it more visible please! (maybe around line 63)

e.g. 
```
..note:: 

If a boolean tensor is provided for any of the mask arguments, positions with a ``True`` value are not allowed to participate in the attention, which is the opposite of the definition for :attr:`attn_mask` in :func:`torch.nn.functional.scaled_dot_product_attention`
```

Reply:
Thanks for the suggestion. Moved the note right below the forward function since the mask args are for the forward function in the transformer.